Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla52487.cs

```diff
@@ -0,0 +1,1028 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+using System.Linq;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System;
+using System.Diagnostics;
+using System.Collections.Generic;
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Threading;
+
+#if UITEST
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(
+		IssueTracker.Bugzilla,
+		52487,
+		"ListView with Recycle + HasUnevenRows generates lots (and lots!) of content view",
+		// https://bugzilla.xamarin.com/show_bug.cgi?id=52487
+		PlatformAffected.iOS
+	)]
+	public class Bugzilla52487 : TestContentPage
```

Comment:
XTC Nunit doesn't like our non-existent .ctor... Maybe the linker removed the default .ctor in the XTC case but not the VS case. Or maybe VS launches nunit in a different way than XTC.